pom.xml

```diff
@@ -116,11 +121,49 @@
 			<groupId>net.sf.jasperreports</groupId>
 			<artifactId>jasperreports-fonts</artifactId>
 			<version>${version.jasperreports}</version>
+			<scope>test</scope>
 		</dependency>
 		<dependency>
 			<groupId>net.sf.jasperreports</groupId>
 			<artifactId>jasperreports-functions</artifactId>
 			<version>${version.jasperreports}</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>net.sf.jasperreports</groupId>
+			<artifactId>jasperreports-jdt</artifactId>
+			<version>${version.jasperreports}</version>
+		</dependency>
+		<dependency>
+			<groupId>net.sf.jasperreports</groupId>
+			<artifactId>jasperreports-charts</artifactId>
+			<version>${version.jasperreports}</version>
+			<scope>test</scope>
+		</dependency>
+
+		<dependency>
+			<groupId>net.sf.jasperreports</groupId>
+			<artifactId>jasperreports-barbecue</artifactId>
+			<version>${version.jasperreports}</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>net.sf.jasperreports</groupId>
+			<artifactId>jasperreports-javascript</artifactId>
+			<version>${version.jasperreports}</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>net.sf.jasperreports</groupId>
+			<artifactId>jasperreports-hibernate</artifactId>
+			<version>${version.jasperreports}</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>net.sf.jasperreports</groupId>
+			<artifactId>jasperreports-google-maps</artifactId>
+			<version>${version.jasperreports}</version>
+			<scope>test</scope>
```

Comment:
Having these jasper dependencies as test scoped, forces the user to include them manually when configuring the plugin. This is a change compared to 3.x of the plugin, where every lib was included, since they where a part of the main jasper library.

I understand the motivation for not including everything from the get go. Excluding and overwriting specific libraries with own versions gets harder, if not downright impossible.

For this PR, I would however prefer to keep the old behaviour, and just include every jasper library as compile scope. We can later revisit this decision, e.g. by providing an additional -min version (without jasper libraries) of the plugin.

WDYT?

Reply:
Since this is "only" a maven plugin, the dependencies wont be included in the build artefact anyway. So from my point of view, these can all be in compile scope.

Reply:
yes , i agree , i forgot that for a maven-plugin dependencies are not included in the build artifact